src/pages/distroSettings/tabs/HostTab/getFormSchema.tsx

```diff
@@ -157,12 +190,127 @@ export const getFormSchema = ({
         arch: {
           "ui:allowDeselect": false,
         },
+        setupAsSudo: {
+          "ui:elementWrapperCSS": css`
+            display: flex;
+            justify-content: flex-end;
+            margin-bottom: 0;
+          `,
+        },
+        workDir: {
+          "ui:description":
+            "Absolute path in which the agent run tasks on the host machine",
+        },
         setupScript: {
+          "ui:elementWrapperCSS": css`
+            margin-top: -22px;
+          `,
           "ui:widget": "textarea",
         },
+        userSpawnAllowed: {
+          ...(hasStaticProvider && {
+            "ui:disabled": true,
+            "ui:tooltipDescription": "Static distros are not spawnable.",
+          }),
+          "ui:description":
+            "Allow users to spawn these hosts for personal use.",
+          "ui:bold": true,
+        },
+        isVirtualWorkStation: {
+          "ui:description":
```

Comment:
it seems this should also be disabled if the arch is [not Linux](https://github.com/evergreen-ci/evergreen/blob/dfb07d9e59ef9db396fdb2e55a3660259b430667/service/templates/distros.html#L1065) 😞